dotCMS/src/main/webapp/WEB-INF/messages/Language.properties

```diff
@@ -4802,5 +4802,8 @@ Default-Action-Archive = Archive
 Default-Action-Unarchive = Unarchive
 Default-Action-Delete = Delete
 Default-Action-Destroy = Destroy
+es.index.read.only.message = At least one of the Elasticsearch current indices are in read only mode
```

Comment:
"is" in read only mode

Reply:
Use this message: Either "Live" or "Working" indices are in read-only mode

Reply:
done https://github.com/dotCMS/core/pull/18604/commits/fc979e0e1d04475951dbb213b379b17022474a8e#diff-f9fa0bd825fbf1166d20a7e5348c93c3R4805